en,zh: use variable to indicate clinic version (#2389)
pingcap/docs-tidb-operator: en/clinic-user-guide.md

To install Helm and configure the chart repository `https://charts.pingcap.org/` maintained by PingCAP, you can refer to the [Use Helm](tidb-toolkit.md#use-helm) document.

> **Note:**
>

> > In the following sections, `${chart_version}` indicates the version of the Diag chart, for example, `v1.3.1`. You can get the currently supported versions through the `helm search repo -l diag` command.

Reviewer: ```suggestion
> In the following sections, `${chart_version}` refer to the version of the Diag chart, for example `v1.3.1`. You can get a list of the currently supported versions by executing the `helm search repo -l diag` command.
```
